.github/workflows/build.yaml

```diff
@@ -21,7 +21,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions-rs/toolchain@v1
         with:
-          components: rustfmt
+          components: rustfmt, clippy
```

Comment:
This lint is called `Lint / Formatting` but adding `clippy` to it changes its purpose.
I think we should ether rename the lint to `Lint / Code` or separate clippy into its own job.

Reply:
A good catch -- I thought (by mistake) that it was both lint and formatting, so I thought I didn't need to change the name. Now, after your remark, I realize that it is a group and a subgroup

Fixed